### PR TITLE
pman OpenShift template for local testing

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
@danmcp This is a WIP template to test pman on local installation of OpenShift. Changes:

- Removed dependency on Swift.
- Used hostPath for writing to host machine's disk(which could be used for pfioh later).

Please let me know, what you think of these changes.

### CODE REVIEW

**Comment by ravisantoshgudimetla** [comment]
@danmcp As per our offline discussion, I added the hostPath and parameters for the existing OpenShift template to work on localhost(to remove dependency on Swift). PTAL. I will update the README once you are fine with these changes.

File: openshift/pman-openshift-template.json [hunk 1]
```diff
@@ -68,9 +68,13 @@
                     "name": "swift-credentials",
                     "mountPath": "/etc/swift/",
                     "readOnly": true
+                  },
+		  {
```
Comment (danmcp) [inline openshift/pman-openshift-template.json]: Something is wrong with the spacing here.  I am also not sure why you are adding a hostpath.  We should probably just talk this through.

**Comment by ravisantoshgudimetla** [comment]
Writing this for my reference and for others to have context.

@danmcp and I had a discussion on the changes and decided it would be better to have two templates, one with swift and other without swift. Previously we wanted to have a single template but the problem is that introduction of hostPath causes pod to not come to running state. We would get an error like:

`Error creating: pods "pman1-1-" is forbidden: unable to validate against any security context constraint: [spec.containers[0].securityContext.volumes[0]: Invalid value: "hostPath": hostPath volumes are not allowed to be used`

So, it was decided to have 2 different templates, one where security context constraint could be changed(local one, where we have access) and one without those constaints being touched.

File: openshift/README.rst [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
     sudo oc login -u system:admin
     sudo oc create sa robot -n myproject
     sudo oc describe sa robot -n myproject
     sudo oc adm policy add-role-to-user edit system:serviceaccount:myproject:robot -n myproject
     sudo oc describe secret <Ex: robot-token-4vhxc> -n myproject
+    
+    ############################
+    # Changes for using hostPath in container. These are not needed, if you want to use swift as backend storage.
+    mkdir /tmp/share           # Create a directory that could be mounted in container. This is mounted as /shareDir in container.
+    setenforce 0               # Disable selinux so that processes can write and read to/from directories in host.
```
Comment (danmcp) [inline openshift/README.rst]: Let's not do this.  We should be able to set the correct perms on /tmp/share

Comment (ravisantoshgudimetla) [reply openshift/README.rst]: I hope changing labels should be ok. Ref: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/getting_started_with_kubernetes/get_started_provisioning_storage_in_kubernetes

**Pull request merged**